src/components/common/data-row-custom/data-row-custom.scss

```diff
@@ -0,0 +1,5 @@
+.data-row-custom-exclamation-indicator {
```

Comment:
as discussed, need to look at why we need the new local style rather than picking up the global

Reply:
moved styling into typography.scss